.github/workflows/nightly-doc-build.yml

```diff
@@ -50,15 +54,20 @@ jobs:
           ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
           PYFLUENT_START_INSTANCE: 0
           PYFLUENT_LAUNCH_CONTAINER: 1
-          DOCS_CNAME: dev.fluentparametric.docs.pyansys.com
           FLUENT_IMAGE_TAG: ${{ matrix.image-tag }}
 
+      - name: Upload HTML Documentation
+        uses: actions/upload-artifact@v3
+        with:
+          name: HTML-Documentation-tag-${{ env.DOC_DEPLOYMENT_IMAGE_TAG }}
+          path: doc/_build/html
+
+          retention-days: 7
+
       - name: Deploy
-        if: matrix.image-tag == 'v22.2.0'
-        uses: JamesIves/github-pages-deploy-action@v4.4.1
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
```suggestion
        if: matrix.image-tag == 'v23.1.0'
```
We'll update the dev documentation nightly, now for 23.1 Fluent after last week's release.